ui/app/AppLayouts/Communities/popups/CommunityMemberMessagesPopup.qml

```diff
@@ -68,6 +68,7 @@ StatusDialog {
                     chatCommunitySectionModule: root.chatCommunitySectionModule
                     messageStore: root.rootStore.messageStore
                     messageId: model.id
+                    joined: root.rootStore.joined
```

Comment:
How does the store know which community we are trying to display here?

Reply:
Before it was directly accessed from `root.rootStore.mainModuleInst.activeSection.joined`, now this has been pushed up to `root.rootStore.joined`. Anyway, I'll review this `joined` property following `Jo's` message [here](https://github.com/status-im/status-desktop/pull/18314#discussion_r2190653788)